apollo-router/src/plugins/progressive_override/mod.rs

```diff
@@ -209,6 +209,23 @@ impl Plugin for ProgressiveOverridePlugin {
                         })
                         .clone();
 
+                    if !relevant_labels.is_empty() {
+                        u64_counter!(
+                            "apollo.router.usage.override.query",
```

Comment:
Suggest that this prefix is changed to either:
`apollo.router.config.`
or
`apollo.router.supergraph.`

Adding a new prefix `usage` doesn't give any extra info over the similarly generic `config`, but you could argue that me make `config` yaml related things and `supergraph` supergraph related.

Reply:
fixed in [0738dee](https://github.com/apollographql/router/pull/4571/commits/0738deef68141834a781b9c1da76c243b5ce548e)